app/src/main/java/chat/rocket/android/authentication/twofactor/presentation/TwoFAPresenter.kt

```diff
@@ -47,4 +50,15 @@ class TwoFAPresenter @Inject constructor(private val view: TwoFAView,
     fun signup() {
         navigator.toSignUp(navigator.currentServer!!)
     }
+
+    private suspend fun registerPushToken() {
+        // TODO: put it on constructor
```

Comment:
There you go... it's already being called twice? This code could definitely be more generic and don't repeated.